tests/SeedTest.php

```diff
@@ -161,16 +161,37 @@ public function testMerge5()
 
         $this->assertSame($o, $oo);
         $this->assertEquals($oo->foo, 'xx');
+    }
 
+    public function testMerge5b()
+    {
         // and even if multiple objects are found
-        $o = new SeedDITestMock();
+        $o = new ViewTestMock();
         $o->foo = ['red'];
-        $o2 = new SeedDITestMock();
+        $o2 = new ViewTestMock();
         $o2->foo = ['yellow'];
         $oo = $this->mergeSeeds(['foo'=>['xx']], $o, ['foo'=>['green']], $o2, ['foo'=>['cyan']]);
 
         $this->assertSame($o, $oo);
-        $this->assertEquals($oo->foo, ['red', 'xx']);
+        $this->assertEquals(['red', 'xx'], $oo->foo);
+    }
+
+    public function testMerge6()
+    {
+        $oo = $this->mergeSeeds(['4'=>'four'], ['5'=>'five']);
+        $this->assertEquals($oo, ['4'=>'four', '5'=>'five']);
+
+        $oo = $this->mergeSeeds(['4'=>['four']], ['5'=>['five']]);
+        $this->assertEquals($oo, ['4'=>['four'], '5'=>['five']]);
+
+        $oo = $this->mergeSeeds(['x'=>['four']], ['x'=>['five']]);
+        $this->assertEquals($oo, ['x'=>['four']]);
+
+        $oo = $this->mergeSeeds(['4'=>['four']], ['4'=>['five']]);
```

Comment:
I wonder why these don't fail, because we still have that `is_numeric` / `is_int` issue: https://github.com/atk4/core/issues/65 not fixed, so when merging these keys `'4'` it should merge these values.

Reply:
hehe me neitherr. in fact i added 'invalid' to that ticket, so when you have time create a proper case for it.